docs/src/content/docs/index.mdx

```diff
@@ -4,114 +4,137 @@ description: Terrateam documentation
+  <Card title="Team Collaboration" icon="approve-check">
+    Review infrastructure changes with intelligent locking to prevent conflicts.
+    <LinkCard title="Collaboration Features" href="/advanced-workflows/locks-and-concurrency" />
   </Card>
-  <Card title="Get on the same page" icon="approve-check">
-    Terrateam enables effective collaboration among team members by providing a pull request-based workflow. Terrateam allows teams to review and discuss proposed changes before applying them. This ensures that everyone is on the same page and reduces the risk of errors and conflicts.
+  
+  <Card title="Enhanced Security" icon="seti:lock">
+    Fine-grained access controls, policy enforcement, and OIDC authentication.
+    <LinkCard title="Access Control" href="/security-and-compliance/role-based-access-control" />
   </Card>
-  <Card title="Safety and consistency" icon="star">
-    Terrateam incorporates sophisticated tracking mechanisms to store Terraform operational state and ensure the safety and consistency of your infrastructure. It intelligently handles dependencies and locks, preventing concurrent modifications and maintaining a reliable and accurate representation of your infrastructure.
+  
+  <Card title="Ready-to-Use" icon="laptop">
+    Skip building custom pipelines. Terrateam handles plan storage, locks, drift detection, and more.
+    <LinkCard title="Drift Detection" href="/advanced-workflows/drift-detection" />
   </Card>
-  <Card title="Customized to your needs" icon="rocket">
-    Terrateam offers a wide range of configuration options and extensibility points, allowing you to tailor the tool to your specific needs and workflows. You can define custom plan and apply steps, configure notifications and integrations, and leverage Terrateam's powerful features like drift detection, cost estimation, and policy enforcement.
+  
+  <Card title="Multi-Environment" icon="star">
+    Manage dev, staging, and production with consistent workflows.
+    <LinkCard title="Configuration" href="/configuration-reference" />
   </Card>
 </CardGrid>
 
-## Why Build When You Can Leverage?
-Building and maintaining a homegrown Terraform solution can be time-consuming, resource-intensive, and prone to errors. For a safe and effective workflow, you must account for the following:
-- Plan file storage and cleanup
-- Approval steps between Plan and Apply
-- Multiple pull requests with overlapping resources
-- Drift Detection and Reconciliation
-- Failed Apply operations
-- Locks and Unlocks
-- Plan and Apply restrictions
-- Secure cloud integrations
-
-Terrateam offers a battle-tested, feature-rich, and actively maintained solution that you can leverage out of the box. By adopting Terrateam, you can focus on writing Terraform code and managing your infrastructure, while benefiting from the collective experience and best practices of the Terrateam community.
-
 ## Getting Started
```

Comment:
The steps are unclear. Add short explanations for each step to help first-time users understand what each step accomplishes.